### PR TITLE
create addTask function to add to list

### PR DESCRIPTION
**Pull request by Ntare22**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,47 +1,48 @@
+import '@fortawesome/fontawesome-free/js/fontawesome.js';
```
Comment (sinansevgi) [inline src/index.js:1]: - [ ] Everything works great but it seems you missed one of the project requirements. You should create a new JavaScript file for new functionalities. Please extract the new functionalities to a new JavaScript file as requested in the project description 
Please check [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Create%20a%20new%20JavaScript%20file%20for%20the%20new%20functionality.)

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @Ntare22, 

Good job so far!
Everything works perfectly.
There are some issues that you still need to work on to go further but you are almost there!

 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Ntare22) [reply src/index.js]: Hi @sinansevgi.. I actually ended up not using it, I went another way. So, I'm just going to remove it.

Comment (sinansevgi) [reply src/index.js]: Hi @Ntare22, Sorry for the misunderstanding. My comment is not about the fort awesome. I attached my comment to the index.js file and it automatically marked the first line of the file. What I was trying to say is, you should create a new javascript file for all of your CRUD functionalities and import that file to your index.js file.

**Review by dasileker — APPROVED** [review]
## **Hi @Ntare22,**

### _**Nice Job 👌**_ 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**